src/NuGet.Core/NuGet.Protocol/Model/PackageDeprecationReasonEnum.cs

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+#nullable enable
```

Comment:
I find it unusual to have a type defined in an assembly, but never used within it. 
![image](https://github.com/user-attachments/assets/b96a2f88-05c5-46cb-b651-74592eab846a)

I think we should move this higher to where it's going to be used. 
I like the rule of 3 for new public APIs, and this has 1 usage that's not even within the same assembly.

Reply:
Moved the enum and renamed it.